src/datastore/pg-store.ts

```diff
@@ -1056,6 +1057,170 @@ export class PgStore {
     });
   }
 
+  async getMempoolStats({ lastBlockCount }: { lastBlockCount?: number }): Promise<DbMempoolStats> {
+    return await this.sql.begin(async sql => {
+      return await this.getMempoolStatsInternal({ sql, lastBlockCount });
+    });
+  }
+
+  async getMempoolStatsInternal({
```

Comment:
This looks really good... I think you should add an index for `fee_rate` and another for `pruned=false` in the mempool table, though

Reply:
@rafaelcr for the `pruned` column -- didn't we figure out that indexes on booleans (e.g. `canonical`) had no effect?

Reply:
Done!